python/lsst/cp/pipe/ptc/cpPtcSolve.py

```diff
@@ -151,12 +148,26 @@ class PhotonTransferCurveSolveConfig(pipeBase.PipelineTaskConfig,
         dtype=int,
         doc="Degree of polynomial to fit the PTC, when 'ptcFitType'=POLYNOMIAL.",
         default=3,
+        deprecated="This field is no longer used. Will be removed after v30."
+    )
+    doModelPtcRolloff = pexConfig.Field(
+        dtype=bool,
+        doc="Model the roll-off in the PTC turnoff as a exponential decay.",
+        default=False,
+    )
+    maxPtcRolloffDeviation = pexConfig.Field(
+        dtype=float,
+        doc="Maximum percent difference between the model with saturation rolloff and the "
+            "model without to set the PTC turnoff. Only used if doModelPtcRolloff=True "
+            "(default = 0.01).",
+        default=0.01,
     )
     doLegacyTurnoffSelection = pexConfig.Field(
         dtype=bool,
         doc="Use 'legacy' computation for PTC turnoff selection. If set "
             "to False, then the KS test p-value selection will be used instead.",
         default=False,
+        deprecated="This option has been deprecated and will be removed after v31.",
```

Comment:
I think this should be v30.  Also please ticket the deprecations into a removal ticket, and add a `TODO: DM-XXXXX Remove deprecated configs`.